spec/rails_helper.rb

```diff
@@ -27,8 +27,7 @@
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   # config.fixture_path = "#{::Rails.root}/spec/fixtures"
 
-  # See database_cleaner.rb for database cleanup details.
-  config.use_transactional_fixtures = false
+  config.use_transactional_fixtures = true
```

Comment:
Note that this setting has nothing to do with Rails fixtures per se; it just means "wrap every example in a transaction and roll it back" as explained here: https://relishapp.com/rspec/rspec-rails/v/3-7/docs/transactions